models/demos/t3000/falcon40b/demo/demo.py

```diff
@@ -286,14 +286,8 @@ def run_falcon_demo_kv(
 
     synchronize_devices(devices)
 
-    profiler.start(f"initializing_KV_cache")
-    kv_cache_singlelayer = initialize_kv_cache(
-        model_config, configuration, 1, batch_size, max_seq_len, devices
-    )  # only used for compile
-    kv_cache = initialize_kv_cache(model_config, configuration, num_layers, batch_size, max_seq_len, devices)
```

Comment:
Can the initialize_kv_cache function be safely deleted now?

Reply:
Yeah you're right!